### ISSUE TITLE
setting whenever RAILS_ENV

### ISSUE THREAD

**Issue by vcavallo** [dresden-weekly/ansible-rails#63]
If deploying to a staging environment, it is necessary to get the env variable into `whenever` so it can write the crontab properly.

I'll submit a pull request that accomplishes this for me.

**Comment by arBmind** [comment]
You should use the same pattern as the module.

```yaml
shell: >
   {{ profile }}'bundle exec rake db:migrate'
```

profile is used to load all the environment variables of the `app_user` profile. This contains the `DATABASE_URL` and `RAILS_ENV` etc. So it should be preferred.

**Comment by vcavallo** [comment]
If i'm not mistaken, the environment needs to be included in the command line arguments to whenever, as well. See my pull request for explanation:
#64

**Comment by arBmind** [comment]
No, the `RAILS_ENV` environment variable is always used if none is given as a commandline argument.

**Comment by vcavallo** [comment]
I realize it's piped through ansible in the `profile`, but how does it get into the the crontab via `whenever`? From what I read in the whenever gem, when the crontab writing command is called, unless the environment is specified in the `schedule.rb` file (and subsequently provided to this file) then it defaults to production.

**Comment by arBmind** [comment]
Ok, I see your point... yes this is an issue.
Your solution is not the right way.

I add a comment to your pull request.
